src/pluto/operation/horizontal_blur.cr

```diff
@@ -23,15 +23,15 @@ module Pluto::Operation::HorizontalBlur
 
         (0..value).each do
           c_value += channel.unsafe_fetch(r_index).to_i - f_value
-          buffer.unsafe_put(c_index, (c_value * multiplier).to_u8)
+          buffer.unsafe_put(c_index, (c_value * multiplier).clamp(0, 255).to_u8)
```

Comment:
I couldn't find a bit twiddle way of fixing this (and thanks for sharing that other fast gaussian implementation). I went with clamp rather than round as round changes the digests of other images while clamp doesn't. Clamp also probably protects more since a `-0.51` or `255.51` would round in the wrong direction.

Reply:
What do you think about going with `#round` to be more in line with the original implementation?

Reply:
I'd be fine with `#round` right up until we find a new image that breaks it for some inexplicable reason :grin: I can update :+1: